app/scripts/proactive/view/JobVariableView.js

```diff
@@ -12,7 +12,7 @@ define(
 
         template: _.template(jobVariableTemplate),
 
-        model: undefined,
+        model: {'jobVariables': {}, 'jobName':'', 'jobProjectName':'', 'jobDescription':'', 'jobDocumentation':'', 'jobGenericInfos':'', 'errorMessage':'', 'infoMessage' :''},
```

Comment:
If the error was only because model was undefined then you could simply do `model:{}`
If the error was raised because some code tried to access a property of model (jobVariables, etc ) then it's fine like you did it.

Reply:
Yes, the error was because `jobProjectName` and `jobDescription` could be undefined, while they are required in the template html.